mycodo/controllers/controller_trigger.py

```diff
@@ -459,3 +465,11 @@ def receive_infrared_code_broadcast(self, code):
             trigger_function_actions(self.unique_id,
                                      message=message,
                                      debug=self.log_level_debug)
+
+    def pre_stop(self):
+        output_channel = db_retrieve_table_daemon(OutputChannel).filter(
+            OutputChannel.unique_id == self.trigger.unique_id_3).first()
+        self.set_output_duty_cycle(
+            self.trigger.unique_id_2,
+            None,
+            output_channel=output_channel.channel)
```

Comment:
When one clicked "deactivate" on a running pwm method, it just kept doing what its current status was. This stops the output gracefully.